### PR TITLE
Fix bug that caused extension-helpers to not work correctly if pyproject was the only configuration file present

### PR DESCRIPTION
**Pull request by astrofrog**
Without this I was seeing issues in projects after migrating setup.cfg -> pyproject.toml

### CODE REVIEW

**Review by saimn — APPROVED** [review]
LGTM

File: extension_helpers/tests/test_setup_helpers.py [hunk 1]
```diff
@@ -342,3 +342,95 @@ def get_extensions():
...
+    test_pkg = tmpdir.mkdir("test_pkg")
+    test_pkg.mkdir(package_name).ensure("__init__.py")
+
+    simple_pyx = test_pkg.join(package_name, "simple.pyx")
+    simple_pyx.write(
+        dedent(
+            """\
+        def test():
+            pass
+    """
```
Comment (neutrinoceros) [inline extension_helpers/tests/test_setup_helpers.py:367]:
```suggestion
        """\
        def test():
            pass
        """
```

**Review by neutrinoceros — APPROVED** [review]
I was able to reprod the error in the wild and I confirm that the test catches it correctly

File: extension_helpers/tests/test_setup_helpers.py (hunk 1 above)
Comment (astrofrog) [reply extension_helpers/tests/test_setup_helpers.py:367]: As we use black to deal with formatting I won't commit this suggestion for now but will look separately at why the formatting isn't quite right here.

**Pull request merged**